pom.xml

```diff
@@ -326,6 +326,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>commons-validator</groupId>
+                <artifactId>commons-validator</artifactId>
+                <version>1.6</version>
+            </dependency>
```

Comment:
Sorry, but we try to keep the code as independent as possible - the only mandatory dependency is _slf4j_. If you want this code, in then copy the relevant methods into our code (with the necessary adjustments) - I suggest placing them in the `SshdSocketAddress` class (which already has some IPv4 related code...)